dht/range_streamer.cc

```diff
@@ -300,7 +300,7 @@ future<> range_streamer::do_stream_async() {
                 unsigned sp_index = 0;
                 unsigned nr_ranges_streamed = 0;
                 size_t nr_ranges_total = range_vec.size();
-                size_t nr_ranges_per_stream_plan = nr_ranges_total / 10;
+                size_t nr_ranges_per_stream_plan = nr_ranges_total / 10 + 1;
```

Comment:
What if there are 760 ranges (no remainder), won't the estimation be off just the same?

Reply:
Yes, the correct calculation is (nr_rangtes_total + 9) / 10.